### PR TITLE
Change unreachableStrategy to be able to start instant replacement tasks

### PR DESCRIPTION
**Pull request by unterstein**
Test Plan:
sbt test
sbt integration:test

Reviewers: jeschkies, zen-dog, jenkins, ichernetsky, kensipe

Reviewed By: jeschkies, zen-dog, jenkins, ichernetsky, kensipe

Subscribers: kensipe, meichstedt, marathon-dev, marathon-team

JIRA Issues: MARATHON_EE-1590

Differential Revision: https://phabricator.mesosphere.com/D963

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
 ## Changes since 1.4.5
 
+### Overview
+#### Changes around unreachableStrategy
+It is now possible to configure `unreachableStrategy` for apps and pods to instantly* replace unreachable apps or pods. To enable this behavior, you need to configure your app or pod as shown below:
+```
+unreachableStrategy: {
+    "inactiveAfterSeconds": 0,
+    "expunceAfterSeconds": 0
```
Comment (idleyoungman) [inline changelog.md]: s/expunceAfterSeconds/expungeAfterSeconds/

File: changelog.md [hunk 2]
```diff
@@ -1,5 +1,17 @@
...
+}
+```
+**Note**: Instantly means as soon as marathon becomes aware of the unreachable task. By default marathon is notified after 75 seconds by mesos
+  that an agent is disconnected. You can change this duration in mesos by configuring `agent-ping-timeout` and `max_agent_ping_timeouts`.
+
```
Comment (kensipe) [inline changelog.md]: s/`agent-ping-timeout`/`agent_ping_timeout`

**Comment by unterstein** [comment]
thanks for the corrections, updated the docs!

**Comment by unterstein** [comment]
Also I added the docs improvements of @kensipe from https://phabricator.mesosphere.com/D963 🎉

File: changelog.md (hunk 2 above)
Comment (jeschkies) [inline changelog.md]: Just a friendly reminder. This suggesting should not lead to a reject of the PR if you trust Johannes to make this change before the launch :).

**Pull request merged**